app/redhat_insights/appserver/static/javascript/views/setup_configuration.js

```diff
@@ -73,7 +78,30 @@ define(["uuid"], function(uuidlib) {
     return splunk_js_sdk_service;
   };
 
+  async function is_app_configured(splunk_js_sdk){
+    //Method to validate if app is configured
+    var application_name_space = {
+      owner: "nobody",
+      app: app_name,
+      sharing: "app",
+    };
+    try{
+      const service = create_splunk_js_sdk_service(splunk_js_sdk, application_name_space)
+      const configFileName = "app";
+      const stanzaName = "install";
+      
+      const isConfigured = await SplunkHelpers.get_config_value(service, configFileName, stanzaName);
+      if (isConfigured){
+        redirect_to_splunk_app_homepage(app_name);
```

Comment:
Shouldn't this check whether the app is configured? If so, then the redirect is outside of its scope.